README.md

```diff
@@ -128,6 +128,7 @@ The following commands are available where the package is installed:
 
 - `./node_modules/.bin/ota-lint-declarations [service_id]...` to lint declarations
 - `./node_modules/.bin/ota-validate-declarations [service_id]...` to test declarations
+- `./node_modules/.bin/ota [service_id]...` to test services and generate artifacts
```

Comment:
“test services and generate artifacts” is not clear to me. Do we mean “to track all declared documents and store their snapshots and versions in the `./data` folder”? We probably want to mention that Git is mandatory in this case, and add guidance for when the user might prefer to run from source rather than using the CLI (i.e. for config), and information on where the declarations will be read from 🙂

Reply:
It's not only to test services, some users may want to use it to really track services. So I suggest:
```
The following commands are available where the package is installed:

- `./node_modules/.bin/ota-lint-declarations [service_id]`: Lint declarations by auto formatting content.
- `./node_modules/.bin/ota-validate-declarations [service_id]` Validate declarations.
- `./node_modules/.bin/ota-track [service_id]` Track services. Recorded snapshots and versions will be store in the `./data` folder at the root of the project where the package is installed.
```